src/CSS/Create.scss

```diff
@@ -23,6 +23,15 @@ h2{
   align-items: center;
 }
 
+#cancelButton {
+  color:#d04c55;
+  width: 5em;
+  height: 2.6em;
+  background-color: #efdccc;
+  border-color: #d04c55;
+  margin-top: 20%;
```

Comment:
don't use % for margins: if your container size increases (e.g. you just add an element in the div), the design will break. prefer rem:

    margin-top: 1rem;

if you need more than 4rem, that's a layout problem: use flexbox/grid ;)